.pre-commit-config.yaml

```diff
@@ -64,6 +64,13 @@ repos:
       - id: black
         name: Format code
 
+  - repo: https://github.com/asottile/blacken-docs
+    rev: v1.10.0
+    hooks:
+      - id: blacken-docs
+        args: [ --line-length=120 ]
+        additional_dependencies: [ black==21.7b0 ]
```

Comment:
that's really cool
is blacken-docs enough frequently maintained for us to adapt it?

Reply:
I would say when it starts to be a problem we just drop it...